compareSVsTSVs

```diff
@@ -152,10 +147,10 @@ class Line(val fields: List[String]) {
           requireConsistentNonRawGeneSets(idx, this.fields(idx), that.fields(idx))
 
         } else if (fusionExpressionColumnIndices.contains(idx)) {
-          // For now the fusion-expression-valued columns of fusion pairs are ignored. Testing them like here
-          // (with much less control over the input than in a unit-test) would mean re-implementing the
-          // feature (gene-name extraction).
-          throw new NotImplementedError()
```

Comment:
why is this deleted? Is the comment reason enough for not having any code at this position?